src/screens/admin/CommunityMemberAdmin.jsx

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import DashboardLayout from '../../layout/dashboardLayout/DashboardLayout'
+import {  getApi, getCommunity, putApi } from '../../utils/apiFunc'
+import './CommunityMemberAdmin.scss';
+
+const ComMemberAdmin = () => {
+    const [data, setData] = useState([]);
+    const [success, setSuccess] = useState(null);
+    
+    const dispatch = useDispatch();
+    
+    // getting current community from localstorage
+    const currentCommunity = getCommunity();
+
+    useEffect(() => {
+        getMemberDetails()
+    }, [success])
+
+    const getMemberDetails = async () => {
+        const { data } = await getApi(
+        dispatch,
+        `${process.env.REACT_APP_API_BASE_URL}/api/communities-users/community/${currentCommunity.id}/details`
+        )
+        setData(data.results)
+    }
+
+    const allowAccess = async (id) => {
+        // const {data} = await axios.put(`${process.env.REACT_APP_API_BASE_URL}/api/communities-users/${id}/community/${currentCommunity.id}`,
+        // {role: "manager"}, config)
+        const {data} = await putApi(
+            dispatch,
+            `${process.env.REACT_APP_API_BASE_URL}/api/communities-users/${id}/community/${currentCommunity.id}`,
+            {role: 'manager'}
+        )
+        setSuccess(data)
+    }
+
+    return (
+        <DashboardLayout title={currentCommunity.name}>
+            <div className="com-member-container">
+                <div className="com-member-header">
+                {
+                    ["firstName", "lastName", "email", "phone", "dateOfBirth", "role", "options"].map(el => {
```

Comment:
do manual lint fix
whole file needs to fix linting seems like its using 4 spaces instead of 2